test/ruby_document_test.rb

```diff
@@ -536,6 +536,52 @@ def test_cache_set_and_get
     assert_equal(value, document.cache_get("textDocument/semanticHighlighting"))
   end
 
+  def test_no_sigil
+    document = RubyLsp::RubyDocument.new(
+      source: +"# frozen_string_literal: true",
+      version: 1,
+      uri: URI("file:///foo/bar.rb"),
+    )
+    value = false
+
+    assert_equal(value, document.sigil_is_true_or_higher?)
```

Comment:
We can drop the `value` and shorten these to:

```
assert(document.sigil_is_true_or_higher?)
# or
refute(document.sigil_is_true_or_higher?)
```
```

Reply:
I forgot about that, let's enable the cop: https://github.com/Shopify/ruby-lsp/pull/1249